app/views/external_users/claims/basic_fees/_basic_fee_fields_primary.html.haml

```diff
@@ -0,0 +1,44 @@
+.form-group.basic-fee-group.nested-fields.fee-details.js-block{class: to_slug(f.object.description), data:{type:'gradFees', autovat: @claim.apply_vat? ? "true" : "false", "block-type": f.object.calculated? ? "FeeBlock" : "FeeBlock"}}
+  %fieldset
+    .form-group
+      %span.bold
+        Case type
+      %p
+        = f.object.claim.case_type.name
```

Comment:
This has 3 levels of indirection, so there's a chance it just 💥 if any of this information does not exist.

At least to safeguard it this should either be using a presenter or something like this:

```
= f.object.claim&.case_type&.name
```

⚠️ This, and the section below for the offences, also suggest that this information is/should be related directly with the **claim** rather than the **fee**.